bin/migration-scripts/upgrade-to-lts.sh

```diff
@@ -296,6 +298,12 @@ function helm2_to_3 {
 function interactive_confirmation {
   echo
   echo
+  echo "Is tiller in the namespace $TILLER_NAMESPACE ?"
+  read -r -p "(y/n) " CONT
+  if ! [ "$CONT" = "y" ]; then
+    echo "Please modify the TILLER_NAMESPACE environment variable at the top of the script."
```

Comment:
Instead of asking people to edit the script, we should handle the custom namespace through either an env var, a command like arg, or both.